about.css

```diff
@@ -0,0 +1,266 @@
+.background-aboutPage {
+  background: url(./assets/img/about-banner.png);
+  background-position: center center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-color: #f7f7f7;
+  width: 100%;
+  max-width: 1300px;
+  margin: auto;
+  height: 620px;
+  display: flex;
+}
+
+.intro-header-container-aboutPage {
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.image-title-aboutPage {
+  width: 60%;
+  margin-top: 90px;
+}
+
+.intro-text-aboutPage {
+  border: 1px solid lightgray;
+  width: 75%;
+  padding: 29px;
+  margin-top: 25px;
+  background-color: #fff;
+  margin-bottom: 35px;
+}
+
+.intro-text-aboutPage span {
+  color: gray;
+  font-size: 13px;
+  font-weight: 900;
+}
+
+.text-aboutPage {
+  font-weight: 900;
+  font-size: 14px;
+  color: #545454;
+  margin: 0 6px;
+}
+
+.emailto {
+  color: black;
+  font-weight: 900;
+}
+
+.background-MainProgram-aboutPage {
+  background-color: #fff;
+  width: 100%;
+  max-width: 1300px;
+  margin: auto;
+}
+
+.mainProgram-container-title-h1-aboutPage {
+  color: black;
+  font-size: 20px;
+  font-weight: 900;
+}
+
+.mainProgram-p-aboutpage {
+  text-align: center;
+  font-size: 12px;
+  font-weight: 600;
+}
+
+.winning-logo {
+  margin: 35px 0;
+}
+
+.winning-logo img {
+  border: 0.1px solid #ececec;
+}
+
+.border-top {
+  border-top: 1px solid lightgray;
+}
+
+.mainProgram-container-title-h1-summit {
+  color: black;
+  font-size: 20px;
+  font-weight: 900;
+  text-shadow: 0.1px 0 black;
+}
+
+.hr_submit_logo {
+  width: 7% !important;
+}
+
+.pastSubmits_container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: 50px 0;
+}
+
+.background-past1 {
+  background-image: url(./assets/img/past1.jpg);
+  width: 275px;
+  height: 135px;
+  background-position: center center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  margin: 0 10px;
+}
+
+.background-past2 {
+  background-image: url(./assets/img/past2.jpg);
+  width: 275px;
+  height: 135px;
+  background-position: center center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  margin: 0 10px;
+}
+
+.color-background-past {
+  background-color: rgba(141, 49, 39, 0.7);
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.pastSubmits-Year {
+  font-size: 23px;
+  font-weight: 900;
+  color: #e5e5e5;
+}
+
+.pastSubmits-location {
+  font-size: 12px;
+  color: #e5e5e5;
+  text-shadow: 0.1px 0 #e5e5e5;
+}
+
+@media only screen and (max-width: 768px) {
+  .intro-title-aboutPage {
+    font-size: 23px !important;
```

Comment:
- [ ] As requested by the previous code reviewer, which is also a best practice, please avoid the use of !important rule, the reason your styles are overridden is because you are placing media queries in front of your normal css styles. This [article ](https://stackoverflow.com/questions/27443589/how-do-i-avoid-using-important) should help you out. Please kindly fix that. As proposed by the previous code reviewer, place media queries at the bottom of your stylesheet.

Reply:
I have addressed this issue, and you were right the usage of the media queries in front of my normal CSS style was affecting my styles and forcing me to use "!important".

I was using 30 "!important" rules in each file.

**Now I have 5 in the whole project!!**

The reason I am using this rule is because of the display none, display block I am using to display different things in my mobile and desktop version.

Thanks!